src/private/runApi.ts

```diff
@@ -144,12 +130,14 @@ async function runApi(
         : requestType.piece.list
       return Promise.resolve(result)
     }
-    const isMock = typeof options.mockResponse !== 'undefined'
+    const isMock = typeof options.mockFetch !== 'undefined'
     options.debugRequests && requestLogger(url, method, {}, isMock)
     requestAction(request)
-    const [error, response] = await attempt(() =>
-      options.mockResponse ? mock(options) : fetch(url, config),
-    )
+
+    if (options.mockTimeout) {
```

Comment:
Note that the `mockTimeout` use is now orthogonal to the use of `mockFetch`. So I could introduce a timer on real fetch calls, it could be useful for troubleshooting.